controllers/controllers.go

```diff
@@ -0,0 +1,27 @@
+package controllers
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+)
+
+func sendJSON(w http.ResponseWriter, data interface{}) {
+	jsData, err := json.Marshal(data)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.Write(jsData)
+}
+
+func toJSON(r *http.Request, v interface{}) error {
+	body, err := ioutil.ReadAll(r.Body)
```

Comment:
Wouldn’t this be fromJSON since going from json to strict?

Reply:
Yes, true. Might name it `bindJSON`